tests/v2/actions/workloads/deployment/deployment.go

```diff
@@ -23,12 +18,7 @@ import (
 )
 
 const (
```

Comment:
missing from this pr that is in others:
```
        port                = "port"
	DeploymentSteveType = "apps.deployment"
```

Reply:
I've rebased again,  but for some reason in the release/v2.7, these constants are not on the deployment.go file, actually they are on https://github.com/rancher/rancher/blob/release/v2.7/tests/v2/actions/workloads/deployment/create.go#L10